omim2obo/main.py

```diff
@@ -197,28 +199,27 @@ def omim2obo(use_cache: bool = False):
         else:
             pass
 
-        if use_abbrev_over_label and abbrev:
-            graph.add((omim_uri, RDFS.label, Literal(abbrev)))
+        if use_abbrev_over_label and pref_symbols:
+            if len(pref_symbols) > 1:
+                raise RuntimeError(f'Unexpected multiple symbols for {omim_id}: {pref_symbols}')
+            graph.add((omim_uri, RDFS.label, Literal(pref_symbols[0])))
         else:
-            graph.add((omim_uri, RDFS.label, Literal(label_cleaner.clean(label))))
+            graph.add((omim_uri, RDFS.label, Literal(label_cleaner.clean(pref_title))))
 
         # Reify on abbreviations. See: https://github.com/monarch-initiative/omim/issues/2
-        exact_labels = [s.strip() for s in label.split(';')]
-        if len(exact_labels) > 1:
-            abbr = exact_labels.pop()
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(abbr)))
-            # Acronym: uppercase, no numbers, no whitespace, <10 chars
-            if abbr.isupper() and not any(char.isspace() for char in abbr) and len(abbr) < 10:
-                axiom = BNode()
-                graph.add((axiom, RDF.type, OWL.Axiom))
-                graph.add((axiom, OWL.annotatedSource, omim_uri))
-                graph.add((axiom, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
-                graph.add((axiom, OWL.annotatedTarget, Literal(abbr)))
-                graph.add((axiom, oboInOwl.hasExactSynonym, MONDONS.abbreviation))
-        for exact_label in exact_labels:
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(exact_label, abbrev))))
-        for label in other_labels:
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(label, abbrev))))
+        for abbreviation in pref_symbols:
+            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(abbreviation)))
+            axiom = BNode()
+            graph.add((axiom, RDF.type, OWL.Axiom))
+            graph.add((axiom, OWL.annotatedSource, omim_uri))
+            graph.add((axiom, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
+            graph.add((axiom, OWL.annotatedTarget, Literal(abbreviation)))
+            graph.add((axiom, oboInOwl.hasExactSynonym, MONDONS.abbreviation))
+        # todo: .clean() should accept all symbols/abbrevs (preferred, alt, included), else [] instead of str/None
+        #  - See: https://github.com/monarch-initiative/omim/issues/129
+        abbrev: Union[str, None] = [None] if not pref_symbols else pref_symbols[0]
```

Comment:
What is this line here doing?

Reply:
`[None]` is not correct and I changed to `None`.

Basically, `abbrev` gets passed to `.clean_label()`, and until I tackle #129, it doesn't accept an empty list for that arg; it expects `None`. So if there are no symbols I convert to `None` and pass that instead.